src/components/JoinCommunity/index.jsx

```diff
@@ -22,6 +22,11 @@ export default function Community() {
                 <p>Join the ever-growing uptane community for updates, support, and collaboration.</p>
             </div>
             <div className={styles.community_links}>
+                <div className={styles.links}>
+                    {/* <LinkCard link="#" title="Slack" light="img/community_links/slack.svg" dark="img/community_links/slack.svg" /> */}
+                    <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="img/community_links/discord-mark-blue.svg" dark="img/community_links/discord-mark-blue.svg" />
+                    {/* <LinkCard link="#" title="Twitter" light="img/community_links/twitter.svg" dark="img/community_links/twitter.svg" /> */}
+                </div>
                 <div className={styles.links}>
                     {/* <LinkCard link="#" title="Slack" light="img/community_links/slack.svg" dark="img/community_links/slack.svg" /> */}
```

Comment:
add the new link under this only, which is `   <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="img/community_links/discord-mark-blue.svg" dark="img/community_links/discord-mark-blue.svg" />` no need of that extra div